#request
uint8 payload_index
uint8 multiplier
float32 x
float32 y
---
#response
bool result